erpc/projects_test.go

```diff
@@ -472,3 +473,101 @@ func TestProject_LazyLoadNetworkDefaults(t *testing.T) {
 		}
 	})
 }
+
+func TestProject_NetworkAlias(t *testing.T) {
+	t.Run("NetworkAliasResolution", func(t *testing.T) {
+		util.ResetGock()
+		defer util.ResetGock()
+		util.SetupMocksForEvmStatePoller()
+		defer util.AssertNoPendingMocks(t, 0)
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		ssr, err := data.NewSharedStateRegistry(ctx, &log.Logger, &common.SharedStateConfig{
+			Connector: &common.ConnectorConfig{
+				Driver: "memory",
+				Memory: &common.MemoryConnectorConfig{
+					MaxItems: 100_000,
+				},
+			},
+		})
+		if err != nil {
+			panic(err)
+		}
+
+		rateLimitersRegistry, err := upstream.NewRateLimitersRegistry(
+			&common.RateLimiterConfig{},
+			&log.Logger,
+		)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		prjReg, err := NewProjectsRegistry(
+			ctx,
+			&log.Logger,
+			[]*common.ProjectConfig{
+				{
+					Id: "prjA",
+					Networks: []*common.NetworkConfig{
+						{
+							Architecture: common.ArchitectureEvm,
+							Evm: &common.EvmNetworkConfig{
+								ChainId: 1,
+							},
+							Alias: "ethereum",
```

Comment:
we shoud have another test where we provide the same alias to two different chains. I don't see any logic enforcing alias uniqueness, but I might jsut be missing it